harvesters/src/main/java/org/fao/geonet/kernel/harvest/HarvestManagerImpl.java

```diff
@@ -79,6 +82,10 @@
 
     private Map<String, AbstractHarvester> hmHarvesters   = new HashMap<String, AbstractHarvester>();
     private Map<String, AbstractHarvester> hmHarvestLookup= new HashMap<String, AbstractHarvester>();
+    
+    private final List<String> summaryHarvesterSettings = Arrays.asList("harvesting","node","site","name","uuid","url","capabUrl","baseUrl","useAccount","ogctype","options","status","info","lastRun");
```

Comment:
Status is probably, but not ogctype. To be checked with all harvester https://github.com/geonetwork/core-geonetwork/tree/develop/harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester